release_notes/ocp_3_10_release_notes.adoc

```diff
@@ -54,33 +54,550 @@ documentation.
 
 This release adds improvements related to the following components and concepts.
 
-[[ocp-310-container-orchestration]]
-=== Container Orchestration
-
 [[ocp-310-storage]]
 === Storage
 
+[[ocp-310-pv-provisioning-using-openstack-manilla]]
+==== Persistent Volume (PV) Provisioning Using OpenStack Manila (Technology Preview)
+
+Persistent volume (PV) provisioning using OpenStack Manila is currently in
+xref:ocp-310-technology-preview[Technology Preview] and not for production
+workloads.
+
+{product-title} is capable of provisioning PVs using the
+link:https://wiki.openstack.org/wiki/Manila[OpenStack Manila] shared file system
+service.
+
+See
+xref:../install_config/persistent_storage/persistent_storage_manila.adoc#persistent_storage_manila[Persistent
+Storage Using OpenStack Manila] for more information.
+
+[[ocp-310-pv-resize]]
+==== PV Resize (Technology Preview)
+
+Persistent volume (PV) resize is currently in
+xref:ocp-310-technology-preview[Technology Preview] and not for production
+workloads.
+
+You can expand persistent volume claims online from {product-title} for CNS
+glusterFS, Cinder, and GCE PD.
+
+. Create a storage class with `allowVolumeExpansion=true`.
+. The PVC uses the storage class and submits a claim.
+. The PVC specifies a new increased size.
+. The underlying PV is resized.
+
+[[ocp-310-ceph-provisioner]]
+==== Ceph Provisioner
+
+WIP
+
+[[ocp-310-CSI]]
+==== Container Storage Interface (Technology Preview)
+
+Container Storage Interface (CSI) is currently in
+xref:ocp-310-technology-preview[Technology Preview] and not for production
+workloads.
+
+CSI allows {product-title} to consume storage from storage backends that
+implement the link:https://github.com/container-storage-interface/spec[CSI
+interface] as
+xref:../architecture/additional_concepts/storage.adoc#architecture-additional-concepts-storage[persistent
+storage].
+
+See
+xref:../install_config/persistent_storage/persistent_storage_csi.adoc#install-config-persistent-storage-persistent-storage-csi[Persistent
+Storage Using Container Storage Interface (CSI)] for more information.
+
+[[ocp-310-Local-Storage]]
+==== Local Storage
+WIP
+
+[[ocp-310-tenant-driven-storage-snapshotting]]
+==== Tenant-driven Storage Snapshotting (Technology Preview)
+
+Tenant-driven storage snapshotting is currently in
+xref:ocp-310-technology-preview[Technology Preview] and not for production
+workloads.
+
+Tenants now have the ability to leverage the underlying storage technology
+backing the persistent volume (PV) assigned to them to make a snapshot of their
+application data. Tenants can also now restore a given snapshot from the past to
+their current application.
+
+An external provisioner is used to access the EBS, GCE pDisk, and HostPath, and
+Cinder snapshotting API. This Technology Preview feature has tested EBS and
```

Comment:
I believe Cinder is not supported at the moment. And QE have tested EBS and GCE pDisk, but not HostPath.

You can got the details from https://trello.com/c/owCRnP8I/521-13-snapshot-implement-the-out-of-tree-snapshotting-controller